clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -823,6 +825,13 @@ synchronized private Config getResourceDescription(ConfigResource resource) {
                 }
                 throw new UnknownTopicOrPartitionException("Resource " + resource + " not found.");
             }
+            case CLIENT_METRICS: {
+                String subscriptionName = resource.name();
```

Comment:
I'd call it `resourceName`. It's not the subscription name - the subscription is the set of metrics which is sent by the broker to a specific client instance, and that's composed of the client metrics config resources which match the client.

Reply:
Make sense, done.